### PR TITLE
Added the log tracing with OpenTelemetry support

### PR DESCRIPTION
**Pull request by Rashmika998**
Added the log tracing with OpenTelemetry support for WSO2 APIM.

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * This class used to generate log telemetry tracer related logs.
+ */
+public class LogExporter implements SpanExporter {
+
+    private final Log log;
+    private final JsonFactory jsonFactory = new JsonFactory();
+
+    public LogExporter() {
+
+        this.log = LogFactory.getLog(TelemetryConstants.TRACER);
```
Comment (tharindu1st) [inline components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java:44]: make declaration and initialization inline.

File: components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    public static LogExporter create() {
+
+        return new LogExporter();
+    }
+
+    @Override
+    public CompletableResultCode export(Collection<SpanData> spans) {
+
+        Iterator var3 = spans.iterator();
+        StringWriter writer;
```
Comment (tharindu1st) [inline components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java:56]: merge to line 60

File: components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        JsonGenerator generator;
```
Comment (tharindu1st) [inline components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java:57]: merge with line 61

**Review by tharindu1st — APPROVED** [review]

File: components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java (hunk 1 above)
Comment (Rashmika998) [reply components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java:44]: Noted and resolved after discussion.

File: components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java (hunk 2 above)
Comment (Rashmika998) [reply components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java:56]: Resolved after discussion.

File: components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java (hunk 3 above)
Comment (Rashmika998) [reply components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/telemetry/LogExporter.java:57]: Resolved after discussion.

**Review by tharindu1st — APPROVED** [review]

**Pull request merged**